tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -49,6 +49,9 @@ export class Ide {
                 Logger.warn('StaleElementException occured during waiting for IDE. Sleeping for 2 secs and retrying.');
                 this.driverHelper.wait(2000);
                 await this.driverHelper.waitAndSwitchToFrame(By.css(Ide.IDE_IFRAME_CSS), timeout);
+            } else {
+                Logger.error(`Switching to IDE frame failed after ${timeout} timeout.`)
```

Comment:
In this case it may be any type of error not mandatory timeout. It may be triky.
I guess this line will be honest only if use not "else" statement but "if (err instanceof error.TimeoutError) {
"

Reply:
Good point. I'll fix that there and also in master.